src/fparser/Fortran2003.py

```diff
@@ -176,7 +178,11 @@ def init(self, *items):
         self.items = items
         return
     def torepr(self):
-        return '%s(%s)' % (self.__class__.__name__, ', '.join(map(repr,self.items)))
+        return '%s(%s)' % (self.__class__.__name__, ', '.join(map(repr,
+                                                                  self.items)))
+    def tostr(self):
+        return self.torepr()
```

Comment:
Yes, tostr() appears to be completely unecessary and I can't remember why I added it. Maybe during debugging. I've deleted it now.